lib/phobos/producer.rb

```diff
@@ -52,6 +52,7 @@ def producer
       class PublicAPI
         NAMESPACE = :phobos_producer_store
         ASYNC_PRODUCER_PARAMS = [:max_queue_size, :delivery_threshold, :delivery_interval].freeze
+        INTERNAL_PRODUCER_PARAMS = [:persistent_connections].freeze
```

Comment:
I know this was my suggestion, but I was just thinking when I see the idea in code if this should be called `SYNC_PRODUCER_PARAMS` 🤔

Either one is fine.

Reply:
I think I'll keep it as INTERNAL in case we want to add additional properties in the future that may or may not be related to sync producing but shouldn't be forwarded to RubyKafka.